### PR TITLE
Checkbox - Migrate to stylable

### PR DESCRIPTION
**Pull request by EgoziE**
Migrating `<Checkbox/>` component styles to Stylable

### CODE REVIEW

File: src/Checkbox/Checkbox.driver.js [hunk 1]
```diff
@@ -3,15 +3,16 @@ import { labelDriverFactory } from 'wix-ui-backoffice/dist/src/components/Label/
 import { testkitFactoryCreator } from 'wix-ui-test-utils/vanilla';
 //TODO - add tooltip classic driver in the correct place
 import { tooltipDriverFactory } from 'wix-ui-core/dist/src/components/tooltip/Tooltip.driver';
-import styles from './Checkbox.scss';
+
 import { dataHooks } from './constants';
+import * as DATA_ATTR from './DataAttr';
```
[the remark this answers is not in the document]
Comment (EgoziE) [reply src/Checkbox/Checkbox.driver.js]: In a different PR
I only migrated to Stylable

File: src/Checkbox/Checkbox.js [hunk 2]
```diff
@@ -47,38 +56,32 @@ class Checkbox extends React.PureComponent {
...
+              ? 'indeterminate'
+              : checked
+              ? 'checked'
+              : 'unchecked',
+            indeterminate,
+            checkboxHover: hover,
+          },
+          this.props,
+        )}
+        data-focus={this.state.isFocused || null}
```
Comment (shlomitc) [inline src/Checkbox/Checkbox.js]: remove

File: src/Checkbox/Checkbox.driver.js [hunk 3]
```diff
@@ -46,11 +46,13 @@ const checkboxDriverFactory = ({ element, eventTrigger }) => {
      * Focus related testing is done in e2e tests only.
      * @deprecated
      */
-    hasFocusState: () => element.getAttribute('data-focus'),
+    hasFocusState: () => element.hasAttribute('data-focus'),
```
Comment (shlomitc) [inline src/Checkbox/Checkbox.driver.js]: remove

File: src/Checkbox/Checkbox.js [hunk 4]
```diff
@@ -47,38 +56,32 @@ class Checkbox extends React.PureComponent {
...
-    */
     return (
       <div
         data-hook={dataHook}
-        className={classname}
-        onFocus={this.props.focusableOnFocus}
-        onBlur={this.props.focusableOnBlur}
-        {...focusableStates(this.props)}
+        {...styles(
+          styles.root,
```
Comment (shlomitc) [inline src/Checkbox/Checkbox.js]: `root`

**Pull request merged**